backends/xnnpack/CMakeLists.txt

```diff
@@ -73,6 +73,27 @@ foreach(fbs_file ${_xnnpack_schema__srcs})
 endforeach()
 
 # Generate the headers from the .fbs files.
+
+if(WIN32)
+  set(MOVE_COMMAND move)
+else()
+  set(MOVE_COMMAND mv)
+endif()
+
+
+if (WIN32)
```

Comment:
this looks fine to me, i'm assuming this has to be done for every flatbuffer outputted header file

Reply:
> this looks fine to me, i'm assuming this has to be done for every flatbuffer outputted header file

Thanks for the review! As suggested, we will split into smaller PRs, and apply these "platform-dependent shell command" changes everywhere in a later PR